src/RoleObjects/MultiLineTextBoxData.js

```diff
@@ -13,11 +12,11 @@ export default class MultiLineTextBoxData extends AccessibilityObject {
     this._reactProps.onSelect = this._onSelect;
   }
 
-  addChild(displayObject) {
+  addChild(displayObject) { // eslint-disable-line
```

Comment:
Please add which rule(s) need to be disabled instead of disabling all linting for this line.